addons/purchase_requisition/migrations/8.0.0.1/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,57 @@
+---Fields in module 'purchase_requisition'---
+purchase_requisition / product.product          / purchase_requisition (boolean): DEL 
+purchase_requisition / product.template         / purchase_requisition (boolean): NEW
+# Field moved
+
+purchase_requisition / purchase.order.line      / quantity_bid (float)          : NEW
+# Duplicated value from current purchase order line quantity
+
+purchase_requisition / purchase.requisition     / account_analytic_id (many2one): NEW relation: account.analytic.account
+purchase_requisition / purchase.requisition     / multiple_rfq_per_supplier (boolean): NEW
+purchase_requisition / purchase.requisition     / ordering_date (date)          : NEW
+purchase_requisition / purchase.requisition     / procurement_id (many2one)     : NEW relation: procurement.order
```

Comment:
This is the counterpart many2one of the existing many2one 'requisition_id' from the procurement to the requisition table. So you can fill this new 'procurement_id' field from here.